src/wazuh_modules/inventory_harvester/qa/test_data/inventory/hardware/005_integrity_check_global/pre_existing_data.json

```diff
@@ -21,7 +21,8 @@
                     "memory": {
                         "free": 1234,
                         "total": 33554432,
-                        "used": 16777216
+                        "used": 16777216,
```

Comment:
The free value does not make any sense if we want representative real values

Reply:
Yes, try to proof the replacement on the `integrity_check` message